gnome-initial-setup/meson.build

```diff
@@ -0,0 +1,30 @@
+gis_assistant_resources = gnome.compile_resources(
+    'gis-assistant-resources',
+    files('gis-assistant.gresource.xml'),
+)
```

Comment:
No dice, but I found a different workaround. I think it's because no symbol from `gis-assistant-resources.c` is explicitly referenced elsewhere (only implicitly) so it is dropped when linking the static library. `export: true` doesn't seem to make any difference in that regard. But when I add an explicit reference in `gis_assistant_class_init` all works fine. I don't understand linkers.

Reply:
Added that patch to the series.